src/slacko.ml

```diff
@@ -335,6 +335,64 @@ type file_obj = {
   num_stars: int option [@default None];
 } [@@deriving of_yojson { strict = false }]
 
+type field_obj = {
+  title: string option [@default None];
+  value: string option [@default None];
+  short: bool option [@default None];
+} [@@deriving yojson { strict = false }]
+
+let field ?title ?value ?short () = {
+  title;
+  value;
+  short;
+}
+
+type attachment_obj = {
```

Comment:
I've successfully posted attachments containing only `fields` and only `text`, so there aren't any specific fields that are mandatory.